include/spm/evt_rpg.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <common.h>
+#include <spm/evtmgr.h>
+#include <evt_cmd.h>
+
+CPP_WRAPPER(spm::evt_rpg)
+
+EVT_DECLARE_USER_FUNC(evt_rpg_effect_check, 1)
```

Comment:
Was this a file we needed to figure out boundaries for?

Reply:
yeah pretty sure its right next to rpgdrv though